src/runtime_src/core/include/xrt/xrt_device.h

```diff
@@ -69,14 +69,22 @@ namespace info {
  *  True if device is a NoDMA device (bool)
  * @var offline
  *  True if device is offline and in process of being reset (bool)
- * @var power_rails
- *  Power rail sensors of device in json format (std::string)
- * @var thermals
- *  Thermal sensors of device in json format (std::string)
- * @power_consumption
- *  Power sensors of a device in json format (std::string)
- * @fans
- *  Fans of a device in json format (std::string)
+ * @var electrical
+ *  Electrical and power sensors present on the device (std::string)
+ * @var thermal
+ *  Thermal sensors present on the device (std::string)
+ * @var mechanical
+ *  Mechanical sensors on and surrounding the device (std::string)
+ * @var memory
+ *  Memory information present on the device (std::string)
+ * @var platform
+ *  Platforms flashed on the device (std::string)
+ * @var pcie_info
+ *  Pcie information of the device (std::string)
+ * @var host
+ *  Host information (std::string)
+ * @var dynamic_regions
+ *  Information about xclbin on the device (std::string)
```

Comment:
I believe these std::strings  are in fact json formatted, right?  If so, maybe state that in the doxygen comments.